mne_bids/tests/test_write.py

```diff
@@ -1544,11 +1543,10 @@ def test_write_anat(_bids_validate):
         rpa=[7.52676800e-02, 0.00000000e+00, 5.58793545e-09],
         coord_frame='head')
 
-    s = 1  # XXX to me it should 1e3 as it shoult be in meters
     mri_scanner_ras_landmarks = mne.channels.make_dig_montage(
-        lpa=np.array([-74.53101434, 19.62855196, -52.28881913]) / s,
-        nasion=np.array([-1.89452708, 103.69850393, 4.97122087]) / s,
-        rpa=np.array([72.01202598, 21.09274957, -57.53678129]) / s,
+        lpa=np.array([-74.53101434, 19.62855196, -52.28881913]),
+        nasion=np.array([-1.89452708, 103.69850393, 4.97122087]),
+        rpa=np.array([72.01202598, 21.09274957, -57.53678129]),
```

Comment:
what is the unit of the coordinates in RAS? if it's a measure of distance then we should
use meters to be consistent with MNE conventions.

Reply:
It's mm so we could convert